### DISCUSSION TITLE
Feature Request: hide manga or anime tabs

### DISCUSSION THREAD

**Discussion by MaxWasUnavailable** [subrotokumar/kurumi#6] [Ideas]
I personally don't read manga. I'm sure there's many others who also only use AniList for anime.

It'd be nice to be able to toggle the manga tab. It would make sense to also offer the option to toggle the anime tab if this were to be implemented.

**Comment by subrotokumar** [comment]
Re: Toggling Anime and Manga Tabs Feature Request

Dear @MaxWasUnavailable,

Thank you for suggesting the idea of introducing toggle options for the anime and manga tabs within our app. We appreciate your input and your enthusiasm for enhancing the user experience.

After careful consideration and taking into account the preferences of our diverse user base, we've decided to maintain the current setup where both anime and manga content are presented together. Our goal is to offer a unified platform that caters to users who enjoy both forms of media with keeping simplicity.

We understand that this decision might not align with your specific preference, and we apologize for any inconvenience. However, we believe that the combined presentation allows us to foster a holistic community and promotes the exploration of various content types.

**Comment by subrotokumar** [comment]
Your request for an anime and manga toggle is like asking for a "make-it-or-break-it" button for our app at the current moment! While we can't promise that just yet, we have a suggestion: start reading more manga! It's like watching anime, but with a little extra imagination and a lot more page-flipping. 😉

In the meantime, we'll keep our gears turning and consider all your feature ideas, including this one. Who knows, maybe one day you'll toggle your way into manga paradise.

Keep flipping.
